master/security/kubernetes-node-ports.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Apply policy to Kubernetes node ports
+---
+
+### Big picture
+
+Restrict access to node ports to specific external clients.
+
+### Value
+
+Exposing services to external clients using node ports is a standard Kubernetes feature. However, if you want to restrict access to node ports to specific external clients, you need to use Calico global network policy.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **GlobalNetworkPolicy** with a preDNAT field
+- **HostEndpoint**
+
+### Concepts
+
+#### Calico global network policy with a preDNAT field 
+
+In a Kubernetes cluster, kube-proxy will DNAT a request to the node's port and IP address, to one of the pods that backs the service. For Calico global network policy to both allow normal ingress cluster traffic, and deny other general ingress traffic, it must take effect before DNAT. To do this, you simply add a **preDNAT** field to a Calico global network policy. The preDNAT field:
```

Comment:
Couple of things:
- It doesn't make any difference to the rendering, but we generally try and keep line lengths under ~100 characters in the docs (unless it's a really big link URL say).  It helps reviewers because they can easily make specific comments, and it also helps people whose editors don't wrap by default.
- I'd lose the second comma.  The first "to ..." is the less important clause compared to the second to.  It's like "Send the train to San Jose, to platform 4".  I'd be tempted to do the same in the second sentence but not too fussed there.

Reply:
Fixed.